data/json/items/ammo.json

```diff
@@ -422,7 +422,7 @@
         "volume" : 1,
         "weight" : 11,
         "ammo_type" : "12mm",
-        "damage" : 45,
```

Comment:
No, we were originally moving ammo types all in one go (update base items, magazines, ammo and spawns) but that seems to have fallen by the wayside and led to partially migrated example such as `12mm`